backstop_data/engine_scripts/puppet/onReady.js

```diff
@@ -11,6 +11,5 @@ governing permissions and limitations under the License.
 */
 
 module.exports = async (page, scenario, vp) => {
-  await require('./hideStuff')(page, scenario);
```

Comment:
Stoked that we don't need to do this!

Reply:
Yes, as when running dev server with `VRT=true gulp dev`. No sidebar will be generated in html.